PSModule/AbstractLauncherTaskCmdlet.cs

```diff
@@ -451,6 +410,47 @@ protected virtual bool CollateResults(string resultFile, string log, string resd
             return true;
         }
 
+        private bool HasResults(string resultFile, out string xmlResults)
+        {
+            xmlResults = null;
+            if (!File.Exists(resultFile))
+            {
+                WriteDebug("result file not found");
+                return false;
+            }
+
+            //read result xml file
+            xmlResults = File.ReadAllText(resultFile);
+
+            if (xmlResults.IsNullOrWhiteSpace())
+            {
+                WriteDebug("Empty results file");
+                return false;
+            }
+            else

```

Comment:
this `else` clause can be omitted as the `if` block always returns `false`.

Reply:
Yes, the `else` clause itself is not functionally important here, just the code from its body.